beginend.el

```diff
@@ -225,16 +225,41 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(add-to-list 'beginend-modes (cons 'mu4e-view-mode-hook #'beginend-message-mode))
```

Comment:
> I would move these values directly into the defvar default value

done

> Emacs might complain about undefined variables

The travis build would fail if that's the case as byte-compilation warnings are interpreted as errors in the Makefile (`byte-compile-error-on-warn`).

> Also format it like so

This indeed looks weird but let's try it.

If you are fine with it, you can merge it :-).